app/controllers/api/v0/highlights_controller.rb

```diff
@@ -48,6 +48,56 @@ def create
     render json: response_binding, status: :created
   end
 
+  swagger_path '/highlights' do
+    operation :get do
+      key :summary, 'Get the highlight(s)'
+      key :description, 'Get the highlight(s) for the owner.'
+      key :operationId, 'getHighlights'
+      key :produces, [
+        'application/json'
+      ]
+      key :tags, [
+        'Highlights'
+      ]
+      # security do
+      #   key :api_id, []
+      # end
```

Comment:
Looks like [swagger 2 doesn't have cookie auth](https://swagger.io/docs/specification/2-0/authentication/) as an option (open API 3 adds it), so I think we can take these out?  Unless you see something to the contrary.